06:42:38.494  root      : INFO    : ++++++++ Tue May 12 06:42:38 2015 ++++++++
06:42:38.601  scapy.runtime: WARNING : No route found for IPv6 destination :: (no default route?)
06:42:38.623  root      : INFO    : Importing platform: eth
06:42:38.623  root      : DEBUG   : Configuration: {'profile': False, 'default_timeout': 2.0, 'platform_args': None, 'xunit_dir': 'xunit', 'disable_ipv6': False, 'random_seed': None, 'interfaces': [], 'port_map': {1: 'veth1', 2: 'veth3', 3: 'veth5', 4: 'veth7'}, 'minsize': 0, 'list_test_names': False, 'openflow_version': '1.3', 'switch_ip': None, 'controller_host': '0.0.0.0', 'default_negative_timeout': 0.01, 'allow_user': False, 'controller_port': 6653, 'test_file': None, 'profile_file': 'profile.out', 'fail_skipped': False, 'test_params': 'None', 'test_dir': '/home/ubuntu/florence/tests-1.3', 'relax': False, 'list': False, 'test_spec': '', 'platform': 'eth', 'log_dir': None, 'xunit': False, 'debug': 'verbose', 'log_file': 'oft.log', 'random_order': False, 'platform_dir': '/home/ubuntu/florence/platforms'}
06:42:38.624  root      : INFO    : OF port map: {1: 'veth1', 2: 'veth3', 3: 'veth5', 4: 'veth7'}
06:42:38.624  root      : INFO    : Autogen random seed: 35136971
06:42:38.627  root      : INFO    : *** TEST RUN START: Tue May 12 06:42:38 2015
06:42:38.627  root      : INFO    : ** START TEST CASE basic.MalformedMessageVersion
06:42:38.628  controller: INFO    : Create/listen at 0.0.0.0:6653
06:42:38.935  controller: INFO    : 0.0.0.0:6653: Incoming connection from ('127.0.0.1', 49652)
06:42:38.935  controller: DEBUG   : Msg out: version 4 class hello len 8 xid 699310543
06:42:38.936  controller: DEBUG   : Msg in: version 4 class hello len 8 xid 134
06:42:38.943  root      : INFO    : Connected ('127.0.0.1', 49652)
06:42:38.943  controller: DEBUG   : Running transaction 1869380712
06:42:38.944  controller: DEBUG   : Msg out: version 4 class features_request len 8 xid 1869380712
06:42:38.944  controller: DEBUG   : Waiting for transaction 1869380712
06:42:38.944  controller: DEBUG   : Msg in: version 4 class features_reply len 32 xid 1869380712
06:42:38.944  controller: DEBUG   : Matched expected XID 1869380712
06:42:38.945  root      : INFO    : Running basic.MalformedMessageVersion
06:42:38.945  controller: DEBUG   : Running transaction 4287098671
06:42:38.946  controller: DEBUG   : Msg out: version 0 class malformed_message len 8 xid 4287098671
06:42:38.946  controller: DEBUG   : Waiting for transaction 4287098671
06:42:38.957  controller: DEBUG   : Msg in: version 1 class bad_request_error_msg len 20 xid 4287098671
06:42:38.957  controller: DEBUG   : Matched expected XID 4287098671
06:42:38.958  root      : INFO    : '\x01\x01\x00\x14\xff\x87\xef/\x00\x01\x00\x00\x00\x00\x00\x08\xff\x87\xef/'
06:42:38.958  root      : INFO    : 1
06:42:38.958  controller: INFO    : Ignoring listen soc shutdown error
06:42:38.959  controller: ERROR   : Unknown socket ready: <socket._socketobject object at 0x7f020363cbb0>
06:42:38.959  controller: INFO    : Exiting controller thread
06:42:38.959  controller: INFO    : Ignoring switch soc shutdown error
06:42:38.959  controller: INFO    : Ignoring listen soc shutdown error
06:42:38.959  root      : INFO    : ** END TEST CASE basic.MalformedMessageVersion
06:42:38.960  root      : INFO    : *** TEST RUN END  : Tue May 12 06:42:38 2015
06:42:38.961  dataplane : INFO    : Thread exit
